### ISSUE TITLE
Issue Loading Saved Resnet50 Model

### ISSUE THREAD

**Issue by PhilBotSeesU** [facebookarchive/caffe2#897]
**Hi, 

I have saved my .mdl file testinit_5.mdl as an example.** 

**I run the following in an effort to load the saved model to begin training it again:** 
python ~/resnet_te/resnet50_trainer.py --train_data /home/ubuntu/resnet_te/data/database/train_db/ --num_gpus=4 --num_epochs=5 --epoch_size=500 --load_model_path=/home/ubuntu/resnet_te/testinit_5.mdl

**But then I get this error message:**

Ignoring @/caffe2/caffe2/contrib/nccl:nccl_ops as it is not a valid file.
Ignoring @/caffe2/caffe2/contrib/gloo:gloo_ops as it is not a valid file.
Ignoring @/caffe2/caffe2/contrib/gloo:gloo_ops_gpu as it is not a valid file.
Ignoring @/caffe2/caffe2/distributed:file_store_handler_ops as it is not a valid file.
Ignoring @/caffe2/caffe2/distributed:redis_store_handler_ops as it is not a valid file.
INFO:resnet50_trainer:Running on GPUs: [0, 1, 2, 3]
INFO:resnet50_trainer:Using epoch size: 480
INFO:data_parallel_model:Parallelizing model for devices: [0, 1, 2, 3]
INFO:data_parallel_model:Create input and model training operators
INFO:data_parallel_model:Model for GPU: 0
INFO:data_parallel_model:Model for GPU: 1
INFO:data_parallel_model:Model for GPU: 2
INFO:data_parallel_model:Model for GPU: 3
INFO:data_parallel_model:Adding gradient operators
INFO:data_parallel_model:Add gradient all-reduces for SyncSGD
INFO:data_parallel_model:Post-iteration operators for updating params
INFO:data_parallel_model:Calling optimizer builder function
INFO:data_parallel_model:Add initial parameter sync
WARNING:data_parallel_model:------- DEPRECATED API, please use data_parallel_model.OptimizeGradientMemory() ----- 
WARNING:memonger:NOTE: Executing memonger to optimize gradient memory
INFO:memonger:Remapping 111 blobs, using 14 shared
INFO:memonger:Memonger memory optimization took 0.0246000289917 secs
WARNING:memonger:NOTE: Executing memonger to optimize gradient memory
INFO:memonger:Remapping 111 blobs, using 14 shared
INFO:memonger:Memonger memory optimization took 0.0234661102295 secs
WARNING:memonger:NOTE: Executing memonger to optimize gradient memory
INFO:memonger:Remapping 111 blobs, using 14 shared
INFO:memonger:Memonger memory optimization took 0.0229349136353 secs
WARNING:memonger:NOTE: Executing memonger to optimize gradient memory
INFO:memonger:Remapping 111 blobs, using 14 shared
INFO:memonger:Memonger memory optimization took 0.0231840610504 secs
INFO:resnet50_trainer:Loading path: /home/ubuntu/resnet_te/testinit_5.mdl
E0630 12:55:23.720163 32740 db.h:234] Usually for a DBReader you should use Read() to be thread safe. Consider refactoring your code.
E0630 12:55:23.736230 32740 db.h:234] Usually for a DBReader you should use Read() to be thread safe. Consider refactoring your code.
INFO:data_parallel_model:Creating checkpoint synchronization net
INFO:data_parallel_model:Run checkpoint net
INFO:resnet50_trainer:Reset epoch to 5

Any help to load a saved model with the resnet50 example would be appreciated!

Thanks
